src/mbgl/util/offscreen_texture.cpp

```diff
@@ -40,10 +43,11 @@ class OffscreenTexture::Impl {
     void bindRenderbuffers(gl::TextureUnit unit) {
         if (!framebuffer) {
             texture = context.createTexture(size, gl::TextureFormat::RGBA, unit);
-            gl::Renderbuffer<gl::RenderbufferType::DepthComponent> depthTarget = context.createRenderbuffer<gl::RenderbufferType::DepthComponent>(size);
-            framebuffer = context.createFramebuffer(*texture, depthTarget);
-
+            gl::Renderbuffer<gl::RenderbufferType::DepthComponent> depth =
+                context.createRenderbuffer<gl::RenderbufferType::DepthComponent>(size);
+            framebuffer = context.createFramebuffer(*texture, depth);
         } else {
+            context.bindTexture(*texture, unit);
```

Comment:
(I read this comment before https://github.com/mapbox/mapbox-gl-native/pull/8896#issuecomment-299428392 and realize it more or less obsoletes this…⏳)